backend/src/main/kotlin/ch/learntrack/backend/admin/grade/GradeQueries.kt

```diff
@@ -0,0 +1,12 @@
+package ch.learntrack.backend.admin.grade
+
+import ch.learntrack.backend.persistence.tables.daos.GradeDao
+import ch.learntrack.backend.persistence.tables.pojos.Grade
+import ch.learntrack.backend.persistence.tables.references.GRADE
+
+public fun GradeDao.isGradeAlreadyExistingInSchool(createGradeDto: CreateGradeDto): Grade? = ctx()
```

Comment:
```suggestion
public fun GradeDao.fetchByNameAndSchoolId(createGradeDto: CreateGradeDto): Grade? = ctx()
```
Or similar. a function name starting with `is` returns a boolen. if you write a function returning a boolen and it isn't called `is` diktat and detekt won't like it.